articles/web-dev's-guide-SEO/index.md

```diff
@@ -0,0 +1,85 @@
+
+## Traditional SEO
+
+Let us look at some of the ways that are used to improve SEO. These are factors responsible for ensuring crawlers can do their job easily. Earlier, the HTML meta tag was used to fill in a set of important keywords. Severe exploitation of the same has resulted in search engines disregarding the keywords. Crawlers are bots that go through pages and find the content present there and list the various links the website directs to. By doing so, crawlers map the internet and enhance the performance of search engines. 
+
+At the same time, the content plays a major role. The amount of time a user spends and the monetization capacity are tracked regularly by the crawlers. Finally, target the niche audience matters. Providing unique content that is designed, delivered, and branded has a higher chance of being ranked higher. The [investment on User Interface and User Experience(UI/UX)](https://neilpatel.com/blog/googlebot-optimization/) returns a larger time spent on the site and more number of clicks resulting in better SEO. 
+
+## PageRank
+The [PageRank](https://en.wikipedia.org/wiki/PageRank) algorithm, named after Larry Page, is the algorithm used to measure and rank the importance of websites. The underlying assumption that ranking algorithms make is as follows: the quality of a site is directly proportional to the number of links to it. Quality links are links to pages that are ranked better. 
+
+Now that we have sufficient background about the topic, let us look at some pro-SEO dev techniques to boost the rankings higher and higher.
+
+### Metadata
+
+1. **Adding meta tag**: Adding meta tags is the oldest way of implementing SEO. These meta tags are placed inside the head tag along with other link tags. Let us look at examples for the same.
+   ```html    
+   <meta name="description" content="Welcome to the blog on SEO on Section.io "/>    
+   <meta name="keywords" content="SEO, Web dev, pagerank, node.js "/>
+    ```
+    The description and keywords are two such options that crawlers look for. Keywords are not used much today, but it's a good practice to include them. 
+
+2. **Adding title tag**: The title tag displays the current title of the webpage. A common mistake that developers make is having the same title tag for most of the pages on a website. The ranking algorithms penalize such behavior of websites for lack of information. The title tag must be modified according to the content at hand. For example, look at the current tab. Is the title " Developer's Guide to Search Engine Optimization| Section"? Such optimization is required for getting better results at search engines.   
```

Comment:
Add the between adding and title.

Remove the space between” and Developer's.

Change results at to results from.